components/at_mention/at_mention.test.jsx

```diff
@@ -164,52 +164,4 @@ describe('components/AtMention', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
-
-    test('should have placement state based on ref position of click handler', () => {
```

Comment:
Can the test be updated instead of deleting?

Reply:
@saturninoabril there was some conversation around this on the initial PR, but I can't recall the specifics around why this was deleted. I believe it relates to difficulties mocking the click handler function. I believe we (previous reviewers) decided to tackle in cypress.